javascript/code-challenges/challenges-09.test.js

```diff
@@ -0,0 +1,308 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function that finds the maximum value in an array
+using the 'reduce' method.
+
+E.g. [4,2,7,5,9,2] -> 9
+------------------------------------------------------------------------------------------------ */
+const maxInArray = (arr) => {
+  let maxVal = arr.reduce((acc, val) =>{
+    return (acc > val ? acc : val);
+  })
+  return maxVal;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named getCourseKeys that takes in the courseInfo object and returns an array containing the keys for the courseInfo object.
+
+For example: (['name', 'duration', 'topics', 'finalExam']).
+------------------------------------------------------------------------------------------------ */
+const courseInfo = { name: 'Code 301', duration: { dayTrack: '4 weeks', eveningTrack: '8 weeks'},
+  topics: ['SMACSS', 'APIs', 'NodeJS', 'SQL', 'jQuery', 'functional programming'],
+  finalExam: true
+};
+
+const getCourseKeys = (obj) => {
+let returnedKeys = Object.keys(obj);
+
+return returnedKeys;
```

Comment:
Watch indentation
also, no need for "returnedKeys" variable. Just return Object.keys(obj)